www/src/components/docs/avatarList.astro

```diff
@@ -46,9 +46,9 @@ const additionalContributors = unique.length - recentContributors.length; // lis
     {
       recentContributors.map((item) => (
         <li>
-          <a href={`https://github.com/${item.login}`}>
+          <a class="rounded-full" href={`https://github.com/${item.login}`}>
             <img
-              class="h-8 w-8 rounded-full sm:h-10 sm:w-10"
+              class="h-8 w-8 sm:h-10 sm:w-10"
```

Comment:
i believe we need the image rounded or overflow-hidden on the a tag:

<img width="342" alt="CleanShot 2023-05-24 at 19 45 17@2x" src="https://github.com/t3-oss/create-t3-app/assets/51714798/e1864544-4526-4def-85f6-6cda07bfcfc8">